src/Group1/g1.java

```diff
@@ -2,7 +2,7 @@
 
 public class g1 {
     public static void main(String[] args) {
-        System.out.println("test16");
+        System.out.println("test105");
         System.out.println("test222");
         System.out.println("test333");
         System.out.println("I am adding something");
```

Comment:
can you  delete line 8 please

Reply:
I deleted

Reply:
![image](https://github.com/Ozleblebici/Batch15_Git_Group_Project/assets/149052999/b6111191-3ed0-4071-8a99-08f3ad007bff)
Test105 is true please update

Reply:
thank you